### ISSUE TITLE
Cannot compile without Jetbrains Annotations

### ISSUE THREAD

**Issue by AlessioDP** [PlaceholderAPI/PlaceholderAPI#401]
## Please read
This template is only for requesting new functions to be added to PlaceholderAPI or for old one to be changed!  
Please use the [Request change (Wiki)] template to ask for additions/changes to the wiki.  
For reporting bugs of PlaceholderAPI use the [Bug report] template.

Also make sure that the [wiki] doesn't already mention such a function/change and that there isn't a issue about this change on the [issues] page.

We recommend to [open a Pull Request][Pull Requests] for making changes to PlaceholderAPI's code.

### Type
> What kind of request is this? (Multiple selections possible)

- [ ] New function for PlaceholderAPI.  
A new function that developers could use.
- [x] Change to code (Internal).  
Changes to code that won't affect the end-user.
- [ ] Change to code (External).  
Changes to code that will affect the end-user (breaks stuff).
- [ ] Other: __________ 

### Info
> What is the change?  
> Please provide as much information (including links, images and code-snippeds) as possible.

I am not able to compile my code due to `@ApiStatus.ScheduledForRemoval(inVersion = "2.10.8")` annotations.

My pom:
```
<dependency>
    <groupId>me.clip</groupId>
    <artifactId>PlaceholderAPI</artifactId>
    <version>2.10.7</version>
    <scope>provided</scope>
</dependency>
```

How I can fix it:
```
<dependency>
    <groupId>me.clip</groupId>
    <artifactId>PlaceholderAPI</artifactId>
    <version>2.10.7</version>
    <scope>provided</scope>
</dependency>
<dependency>
    <groupId>org.jetbrains</groupId>
    <artifactId>annotations</artifactId>
    <version>19.0.0</version>
</dependency>
```

Yesterday I build it successfully thanks to PlaceholderAPI-DEV-66 build but with DEV-75 it doesn't work anymore.

[Request change (Wiki)]: https://github.com/PlaceholderAPI/PlaceholderAPI/issues/new?template=change_request_wiki.md
[Bug report]: https://github.com/PlaceholderAPI/PlaceholderAPI/issues/new?template=bug_report.md
[issues]: https://github.com/PlaceholderAPI/PlaceholderAPI/issues
[wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki
[Pull Requests]: https://github.com/PlaceholderAPI/PlaceholderAPI/pulls

**Comment by PyvesB** [comment]
Hello I'm seeing this as well in my projects.

**Comment by extendedclip** [comment]
@PyvesB Can you test this build in the dev repo?

`  <repository>
    <id>dev</id>
    <url>https://repo.extendedclip.com/content/repositories/dev</url>
  </repository>

<dependency>
  <groupId>me.clip</groupId>
  <artifactId>placeholderapi</artifactId>
  <version>2.10.8-DEV-93</version>
</dependency>`

**Comment by extendedclip** [comment]
Fixed in 098250557401278fd4572e91656f330f7ccb1740

**Comment by PyvesB** [comment]
> 
> 
> @PyvesB Can you test this build in the dev repo?
> 
> ```
> <repository>
>   <id>dev</id>
>   <url>https://repo.extendedclip.com/content/repositories/dev</url>
> </repository>
> 
> <dependency>
>   <groupId>me.clip</groupId>
>   <artifactId>placeholderapi</artifactId>
>   <version>2.10.9-DEV-93</version>
> </dependency>```
> ```

I have tested this locally and can confirm that it works. Thanks for looking into this issue promptly, @extendedclip ! 👍🏻
